lib/index.js

```diff
@@ -3,7 +3,9 @@
  * Module dependencies.
  */
 
+'use strict';
```

Comment:
This should go at the very top of the file, but that's just a nitpick :stuck_out_tongue: